tests/pykafka/test_producer.py

```diff
@@ -96,11 +95,11 @@ def test_async_produce_thread_exception(self):
     def test_async_produce_unicode(self):
         """Ensure that the producer can handle unicode strings"""
         topic = self.client.topics[self.topic_name]
-        payload = u"tester"
+        payload = b"tester"
```

Comment:
This changes the nature of the test (which wants to test with unicode specifically).

(Also, it should probably send some non-ascii chars to really test that.)